otx/cli/tools/train.py

```diff
@@ -250,8 +257,23 @@ def main():  # pylint: disable=too-many-branches
             prediction_dataset=predicted_validation_dataset,
         )
         task.evaluate(resultset)
-        assert resultset.performance is not None
-        print(resultset.performance)
+        performance = resultset.performance
+        assert performance is not None
+        print(performance)
+
+    end_time = time.time()
+    sec = end_time - start_time
+    total_time = str(datetime.timedelta(seconds=sec))
```

Comment:
How about just to use `end - start` without datetime? Do we have special reason to use datetime?

Reply:
It's just because it's easy to see.